engine/src/main/java/org/pentaho/di/trans/steps/getvariable/GetVariable.java

```diff
@@ -140,8 +143,40 @@ public boolean init( StepMetaInterface smi, StepDataInterface sdi ) {
     return false;
   }
 
+  @Override
   public void dispose( StepMetaInterface smi, StepDataInterface sdi ) {
     super.dispose( smi, sdi );
   }
 
+  public JSONObject doAction( String fieldName, StepMetaInterface stepMetaInterface, TransMeta transMeta,
```

Comment:
Map<String, String> queryParams 
The above param has been added to the doAction method
Please rebase with latest feature branch